src/util.c

```diff
@@ -740,6 +745,55 @@ get_random_bytes(uint num) {
 	return buf;
 }
 
+// Legacy naming scheme for virtio block devices
+// Reference : https://github.com/torvalds/linux/blob/master/drivers/block/virtio_blk.c#L478
+static int
+virtblk_name_format(char *prefix, int index, char *buf, int buflen)
+{
+	const int base = 'z' - 'a' + 1;
+	char *begin = buf + strlen(prefix);
+	char *end = buf + buflen;
+	char *p;
+	int unit;
+
+	p = end - 1;
```

Comment:
well, there is a mini mind melting algo - if we'd written it, I'd be asking for an explanatory comment - but, as it is a clone of the 'standard', that's fine.